Apps/AdminWebClient/src/ClientApp/src/services/restCommunicationService.ts

```diff
@@ -37,4 +37,30 @@ export class RestCommunicationService implements ICommunicationService {
         });
     });
   }
+
+  public getBannerList(): Promise<Communication[]> {
+    return new Promise((resolve, reject) => {
+      this.http
+        .get<RequestResult<Communication[]>>(`${this.BASE_URI}`)
```

Comment:
At this point it looks like getBannerList should be getCommunications

Reply:
getCommunications would seem appropriate name.

Reply:
I will update the name.
Do we want to refactor also existing code for constancy with the naming, i.e. "public getFeedbackList(): Promise<UserFeedback[]> {"   ? If yes, I can create a task for it?

Reply:
We could, getCommunicationsList would also work

Reply:
To be consistent with existing code, i.e. getUsers, getNotes, etc.., I have updated the name to getCommunications.